src/node_env_var.cc

```diff
@@ -213,19 +183,19 @@ std::shared_ptr<KVStore> KVStore::Clone(v8::Isolate* isolate) const {
   for (uint32_t i = 0; i < keys_length; i++) {
     Local<Value> key = keys->Get(context, i).ToLocalChecked();
     CHECK(key->IsString());
-    copy->Set(isolate, key.As<String>(), Get(isolate, key.As<String>()));
+    copy->Set(isolate, key.As<String>(), Get(isolate, key.As<String>())
+        .FromMaybe(Local<String>()));
```

Comment:
This should be just `ToLocalChecked()`

Reply:
fixed at https://github.com/nodejs/node/pull/27310/commits/fef9434123fb1eb3aabeab08345f65cfd0f5ce7d

i've used FromMayBe function because https://v8docs.nodesource.com/node-10.6/d8/d7d/classv8_1_1_maybe_local.html doc says, v8 would crash if Maybe<> is an empty handle. But at this points you mentioned in review it is unlikely to be empty. 👍🏻